README.md

```diff
@@ -87,15 +87,15 @@ or
 ```
 
 - Pulling your request. Click on the Pull requests tab on the forked github repository.
```

Comment:
```suggestion
- Navigate to your fork, on the top of the files section you'll notice a new section containing, a contribute button!
- Click on the contribute button, it will open a drop down, click the pull request button on the drop down
```

Reply:
Addition to @kaiwalyakoparkar's suggestion